data/recorder/src/main/java/tech/pegasys/teku/data/recorder/SSZTransitionRecorder.java

```diff
@@ -41,13 +40,12 @@ public SSZTransitionRecorder(final Path outputDirectory) {
   }
 
   @Subscribe
-  public void onGenesis(final UpdatableStore store) {
-    final Checkpoint finalizedCheckpoint = store.getFinalizedCheckpoint();
+  public void onGenesis(final GenesisEvent genesis) {
+    final Checkpoint finalizedCheckpoint = genesis.getCheckpoint();
```

Comment:
Would it be difficult to switch this over to using the EventChannel interface to get this event rather than the EventBus while you're here?  Don't worry if it takes more than a couple of minutes but it would be nice to get rid of one more usage of EventBus.

Reply:
Started looking into this, but I wanted to make a generic `ChainUpdates` channel to consolidate various chain events and decided it was too much to put into this PR.